### PR TITLE
Fix deprecation

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -56,14 +56,14 @@ class BabelEsmPlugin {
...
+          new MultiEntryPlugin(compiler.context, entryFiles, entry).apply(childCompiler);
         } else {
-          childCompiler.apply(new SingleEntryPlugin(compiler.context, entryFiles, entry));
+          new SingleEntryPlugin(compiler.context, entryFiles, entry).apply(childCompiler);
         }
       });
 
       // Convert entry chunk to entry file
-      childCompiler.apply(new JsonpTemplatePlugin());
+      new JsonpTemplatePlugin().apply();
```
Comment (prateekbh) [inline src/index.js]: no `childCompiler` passed to JsonpTemplatePlugin?

Comment (prateekbh) [reply src/index.js]: also the tests suggested that might be needed

**Comment by prateekbh** [comment]
seems to fix it.. @ForsakenHarmony last commit looks good?

**Comment by ForsakenHarmony** [comment]
oh ye, missed that 🙈

**Pull request merged**